tests/cases/Issue89.ts

```diff
@@ -0,0 +1,4 @@
+export class ^FileLogger ^extends /**/^One ^implements/**/ ^IDisposable{
+	^	
+}
+^
```

Comment:
Add more tests for complex cases:
1. `extends` more than one things: `export class A extends B, C, D implements E, F, G {`
2. switch the positions of `extends` and `implements`